### PR TITLE
Fix: using isinstance to check list type

### PR DESCRIPTION
**Pull request by andrewwoood**
Fixes [this issue](https://github.com/alpacahq/alpaca-py/issues/148)

### CODE REVIEW

File: alpaca/trading/client.py [hunk 1]
```diff
@@ -591,7 +591,7 @@ def get_corporate_annoucements(
         """
         params = filter.to_request_fields() if filter else {}
 
-        if "ca_types" in params and type(params["ca_types"]) is List:
+        if "ca_types" in params and params["ca_types"]:
```
Comment (OlegRa) [inline alpaca/trading/client.py:594]: Just a curious (I'm not a Python expert) why did you remove the type check at all in this case? It looks like an error to me - most probably we need the same `isinstance` check here.

Comment (andrewwoood) [reply alpaca/trading/client.py:594]: You're right that this is an error, I should have checked type as you've pointed out. I'll fix this now, thank you 🤝

**Review by haxdds — APPROVED** [review]
